account_invoice_report_due/__openerp__.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+# Manuel Claeys Bouuaert <manuel@coopiteasy.be>
+
```

Comment:
This is not the usual header for copyright. Have a look [here](https://github.com/coopiteasy/addons/blob/12.0/account_invoice_line_update/__manifest__.py#L1) for example